pages/linux/strace.md

```diff
@@ -0,0 +1,16 @@
+# strace 
+
+> Troubleshooting tool for tracing system calls
+
+- Start tracing a specific PID 
+
+`strace -p {{pid}}`
+
+- Trace a process and filter output by system call
+
+`strace -e {{system call name}} -p {{pid}}`
```

Comment:
I'm not familiar with this command; does it still work if the order of the options is switched? If so, I'd suggest to do it, because that would match the description (and the intuitive mental model) better.